src/lib/device.js

```diff
@@ -34,9 +34,12 @@ export async function canDownload() {
   log.debug('device/canDownload');
   let [native, version, ios] = await Promise.all([isNative(), getVersion(), isIos()]);
 
+  // support any development versions like X.Y-dev for compare versions
+  let dashIndex = version.indexOf('-');
```

Comment:
"TypeError: Cannot read property 'indexOf' of undefined" in webview

- [ ] fix undefined error

Reply:
As the version is always, this can't be undefined